src/components/dashboard/dashboardItem/ItemBase.js

```diff
@@ -45,6 +45,45 @@ const DashboardLink = ({ target, kind, children }) => {
     );
 };
 
+function AnalysisSubheader(props) {
+    const { analysis, date: formattedDate } = props;
+    const { t } = useTranslation(["dashboard", "apps"]);
+    const terminated = isTerminated(analysis);
+
+    const [runningTime, setRunningTime] = useState(null);
+
+    useEffect(() => {
+        const handleUpdateRunningTime = () => {
+            setRunningTime(formatDistanceToNow(new Date(analysis.start_date)));
+        };
+
+        let interval;
+        if (!terminated) {
+            interval = setInterval(handleUpdateRunningTime, 60000);
+            handleUpdateRunningTime();
+        }
+
+        return () => clearInterval(interval);
```

Comment:
Also we might want to move some of this logic to some where like `utils` since this is requested in `Analyses` view as well.